modules/virt-cnv-storage-alerts.adoc

```diff
@@ -0,0 +1,377 @@
++
+[source,terminal]
+----
+oc get pods -n <dv_namespace> -o json | jq -r '.items[] | select(.metadata.ownerReferences[] | select(.name=="<dv_name>")).metadata.name'
+----
+
+* Change the `<worker_pod>` field that corresponds to the erroneous DV's namespace to check the failing deployments’ corresponding description and logs.
++
+[source,terminal]
+----
+oc -n <dv_namespace> describe pods <worker_pod>
+
+oc -n <dv_namespace> logs <worker_pod>
+----
+
+.Mitigation
+
+In some cases, the error is an incorrect URL, indicated by a 404 error when diagnosing the problem. If an incorrect URL is the cause, you can restart by deleting the DV, correcting the URL on the DV manifest, and recreating the DV.
+
+Open an issue and provide the information gathered in the diagnosis process.
+
+//CDINotReady Alert
+== CDINotReady alert
+
+.Meaning
+
+If a CDI installation is in a degraded state, then the installation is not progressing or not available to use.
+
+.Impact
+
+If the CDI is unusable, then you cannot build Virtual Machine Disks on PVCs using CDI’s DataVolumes (DVs). Additionally, components are not ready and stop progressing towards a ready state.
+
+.Diagnosis
+
+* Check the cdi-operator’s pod namespace.
++
+[source,terminal]
+----
+export openshift-cnv="$(oc get deployment -A | grep cdi-operator | awk '{print $1}')"
+----
+In this example, the kind of operand is `kubevirt` and the operand's name is `kubevirt-kubevirt-hyperconverged`.
+
+* Verify if any of the CDI components are currently not ready.
++
+[source,terminal]
+----
+oc -n $openshift-cnv get deploy -l cdi.kubevirt.io
+----
+
+* Check the failing deployments’ corresponding pod logs and describe (substitute).
++
+[source,terminal]
+----
+oc -n $openshift-cnv describe pods <corresponding_pod_name>
+
+oc -n $openshift-cnv logs <corresponding_pod_name>
+----
+
+.Mitigation
+
+Open an issue and provide the information gathered in the diagnosis process.
+
+//CDIOperatorDown Alert
+== CDIOperatorDown alert
+
+.Meaning
+
+The CDI Operator is responsible for deploying and managing the CDI infrastructure components such as the DataVolume or PVC controllers that help you build Virtual Machine Disks on PVCs.
+
+This alert fires when the CDI operator is down.
+
+.Impact
+
+If the CDI operator is down, then the dependent infrastructure components may not deploy at all or fail to stay in a required state. As a result, the CDI installation is not fully operational in the cluster.
+
+.Diagnosis
+
+* Check the cdi-operator’s pod namespace.
++
+[source,terminal]
+----
+export openshift-cnv="$(oc get deployment -A | grep cdi-operator | awk '{print $1}')"
+----
+
+* Verify the cdi-operator’s pod is currently down.
++
+[source,terminal]
+----
+oc -n $openshift-cnv get pods -l name=cdi-operator
+----
+
+* Check cdi-operator’s pod description and logs.
++
+[source,terminal]
+----
+oc -n $openshift-cnv describe pods -l name=cdi-operator
+
+oc -n $openshift-cnv logs -l name=cdi-operator
```

Comment:
@mgarrellRH , Lets revert this and keep the $NAMESPACE instead of $openshift-cnv